### ISSUE TITLE
[@types/webpack-chain] Partial function lacks type definition.

### ISSUE THREAD

**Issue by fimars** [DefinitelyTyped/DefinitelyTyped#27519]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/webpackchain` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/). (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @eirikurn @psachs21

If you do not mention the authors the issue will be ignored.

```typescript
import Config = require('webpack-chain');
const config = new Config();

// `config.mode` and `config.parallelism` lacks type definition.
```

**Comment by gluons** [comment]
Any news of this issue?  
This problem completely block me from using `webpack-chain` in TypeScript.  
I have to use `webpack-chain` without declaration.

**Comment by fimars** [comment]
Maybe you can fork the `webpack-chain` and install the fork version by `npm install **git address**`. @gluons

**Comment by edmorley** [comment]
There's no need to fork, just open a PR! :-)

(There is an existing one that was closed by its author for unknown reasons that you could use as a starting point: neutrinojs/webpack-chain#123)

**Comment by gluons** [comment]
Resolved by neutrinojs/webpack-chain#132 🎉
